contrib/kafka/filters/network/test/kafka_response_parser_test.cc

```diff
@@ -73,13 +73,6 @@ TEST_F(KafkaResponseParserTest, ResponseHeaderParserShouldExtractHeaderAndResolv
 }
 
 TEST_F(KafkaResponseParserTest, ResponseHeaderParserShouldThrowIfThereIsUnexpectedResponse) {
-  // given
-  const int32_t payload_length = 100;
-  const int32_t correlation_id = 1234;
-  uint32_t header_len = 0;
-  header_len += putIntoBuffer(payload_length);
-  header_len += putIntoBuffer(correlation_id); // Insert correlation id.
```

Comment:
this test passes, but were these supposed to have an important side effect?

Reply:
Looks fine! I'll revisit this place in a few days.

Reply:
Thanks!